src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Runtime/DoNotDefineFinalizersToTypesDerivedFromMemoryManagerTests.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Testing;
+using Xunit;
+using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.DoNotDefineFinalizersToTypesDerivedFromMemoryManager,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+using VerifyVB = Test.Utilities.VisualBasicCodeFixVerifier<
+    Microsoft.NetCore.Analyzers.Runtime.DoNotDefineFinalizersToTypesDerivedFromMemoryManager,
+    Microsoft.CodeAnalysis.Testing.EmptyCodeFixProvider>;
+
+namespace Microsoft.NetCore.Analyzers.Runtime.UnitTests
+{
+    public class DoNotDefineFinalizersToTypesDerivedFromMemoryManagerTests
+    {
+        [Fact]
+        public async Task ClassNotDerivedFromMemoryManagerOK()
+        {
+            await VerifyCS.VerifyAnalyzerAsync(@"
+using System;
+
+namespace TestNamespace
+{
+    class TestClass
+    {
+        private void TestMethod() { }
+    }
+}");
```

Comment:
Nit: The file is more readable with this code body text indented.

Reply:
For reference, I thought this was easier to read, with the string of code intended to align with the unit test code: https://github.com/dotnet/roslyn-analyzers/pull/3443/files#diff-f2d2a722cf880cf930846c778433d9c4R31-R44